src/components/CommentList.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react"
+import { getCommentsById } from "../api";
+import CommentCard from "./CommentCard";
+
+export default function CommentList({articleId}) {
+
+    const [comments, setComments] = useState([]);
+
+    useEffect(() => {
+        getCommentsById(articleId)
+        .then((data) => {
+            setComments(data)
+        })
+        .catch(() => {
+            console.error(`Failed to fetch Comments`)
+        });  
+    }, [articleId])
+    
```

Comment:
Nice this is good logic, but currently there is no indication that this information is loading. How can we let the user know that this is the case?

Something that is just as important as well is that when navigating to an article with no comments (article id 37) which has no comments, nothing is shown to the user. How might we let the user know that there are no comments for an article like this?